pkg/networkservice/chains/nsmgr/single_test.go

```diff
@@ -254,3 +253,64 @@ func Test_Local_NoURLUsecase(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, int32(1), atomic.LoadInt32(&counter.Closes))
 }
+
+func Test_ShouldParseNetworkServiceLabelsTemplate(t *testing.T) {
+	t.Cleanup(func() { goleak.VerifyNone(t) })
+
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second*5)
+	defer cancel()
+
+	const (
+		testEnvName             = "NODE_NAME"
+		testEnvValue            = "testValue"
+		destinationTestKey      = "nodeName"
+		destinationTestTemplate = `{{index .Src "NodeNameKey"}}`
+	)
+
+	err := os.Setenv(testEnvName, testEnvValue)
+	require.NoError(t, err)
+
+	domain := sandbox.NewBuilder(t).
+		SetNodesCount(1).
+		SetRegistryProxySupplier(nil).
+		SetNSMgrProxySupplier(nil).
+		SetContext(ctx).
+		Build()
+
+	ns := defaultRegistryService()
```

Comment:
So we have `nsReg` and `nseReg` names for the NS and NSE registration variables in all of the rest NSMgr sandbox tests. Please don't introduce some new names for such common things.